src/main/java/kg/peaksoft/giftlistb6/db/services/MailingListService.java

```diff
@@ -0,0 +1,65 @@
+package kg.peaksoft.giftlistb6.db.services;
+
+import kg.peaksoft.giftlistb6.db.models.MailingList;
+import kg.peaksoft.giftlistb6.db.repositories.MailingListRepository;
+import kg.peaksoft.giftlistb6.dto.requests.MailingListRequest;
+import kg.peaksoft.giftlistb6.dto.responses.AllMailingListResponse;
+import kg.peaksoft.giftlistb6.dto.responses.MailingListResponse;
+import kg.peaksoft.giftlistb6.exceptions.NotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import javax.transaction.Transactional;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class MailingListService {
+
+    private final MailingListRepository mailingListRepository;
+
+    public AllMailingListResponse saveMailingList(MailingListRequest request) {
+        MailingList mailingList = convertToEntity(request);
+        mailingListRepository.save(mailingList);
+        return convertToResponse(mailingList);
+    }
+
+    @Transactional
+    public MailingList convertToEntity(MailingListRequest request) {
+        MailingList mailingList = new MailingList();
+        mailingList.setName(request.getName());
+        mailingList.setPhoto(request.getImage());
```

Comment:
Why setPhoto and getImage?

Reply:
entity was created with photo field, ok i edited it